packages/react-components/src/components/xy-output-component.tsx

```diff
@@ -83,6 +84,8 @@ export class XYOutputComponent extends AbstractTreeOutputComponent<AbstractOutpu
 
     private onSelectionChanged = (payload: { [key: string]: string; }) => this.doHandleSelectionChangedSignal(payload);
 
+    private _throttledUpdateXY = throttle(() => this.updateXY(), 500);
```

Comment:
I'm not sure, but if I understand correctly, if you don't put the trailing option then you might miss the last update if it happens within 500ms of the last throttled update?

Reply:
I thought default throttle has settings {leading: true, trailing: true}. Should I explicitly mention {leading: true, trailing: true} or {leading: false, trailing: true} ?

Reply:
OK I see those defaults in the web documentation, but not in the inline Javadoc. So if the web documentation is correct, it's fine as it is.